link/src/main/java/com/stripe/android/link/ui/LinkButtonView.kt

```diff
@@ -90,38 +87,63 @@ fun LinkButton(
                     bottom = LinkButtonVerticalPadding
                 )
             ) {
+                Spacer(modifier = Modifier.weight(1f))
```

Comment:
We don’t need the spacers because the `Button` uses a center-aligning `RowScope` for its content.